news/881.bugfix

```diff
@@ -0,0 +1 @@
+Fix integer/float choice from spec in nevergrad plugin
```

Comment:
I think case we should probably use [plugin](https://hydra.cc/docs/next/development/documentation) file suffix.